src/command_modules/azure-cli-dla/azure/cli/command_modules/dla/_help.py

```diff
@@ -185,6 +205,11 @@
   - name: --allow-azure-ips
     type: string
     short-summary: Allow or block IPs originating from Azure through the firewall.
+examples:
+  - name: Update a Data Lake Analytics account
```

Comment:
should this say autogenerated?

Reply:
Yes, you are absolutely right. I am working on a fix for this.